examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -1,34 +1,38 @@
 """
-=======================================================
-Creating a TimeSeries from GOES-XRS near real time data
-=======================================================
+========================================================================
+Creating a TimeSeries from GOES-XRS near real time data with flare times
+========================================================================
 
-This example will demonstrate how to download and load GOES XRS Near Real Time (NRT) data into a `sunpy.timeseries.TimeSeries`.
+This example will demonstrate how to make use of GOES XRS Near Real Time data.
+This includes the goes XRS timeseries data as well as the flare times.
+The real-time datasets can all be found at https://services.swpc.noaa.gov/json/goes/primary/
 """
 
 import matplotlib.pyplot as plt
 import pandas as pd
 
 from astropy import units as u
+from astropy.time import TimeDelta
```

Comment:
```suggestion
from astropy.time import Time
```
* I need to instantiate `Time` directly because `parse_time()` has a bug
* The use of `TimeDelta` is unnecessary